### ISSUE TITLE
Regression with electron 18: Outlook Web App nativefied shows no attachment preview

### ISSUE THREAD

**Issue by dkadioglu** [nativefier/nativefier#1401]
**Homework**

- [x] I looked at `nativefier --help` and https://github.com/nativefier/nativefier/blob/master/API.md
- [x] I checked the [Troubleshooting section of the README](https://github.com/nativefier/nativefier/blob/master/README.md#troubleshooting).
- [x] I searched existing issues, open & closed. Yes, my bug is new.
- [x] I'm using the latest version available at https://github.com/nativefier/nativefier/releases

**Bug description**

Using the most recent release 47.0.0, attachments of an email are not shown anymore as preview in nativefied Outlook Web App. If I build the nativefied Outlook Web App with --electron-version 17.4.0, attachment preview works again.

**Steps to reproduce**

Give your ***full*** nativefier command and its logs, with the ***`--verbose` flag***, on a ***public*** site:
Build with issue:
Command:
```
nativefier -n "Outlook Web" -p linux -a x64 --tray https://some-owa-server.url --single-instance --internal-urls "(.*?)(xx\.yy\.zz)(.*?)" --icon OutlookWeb.ico --inject OutlookWeb.js -m --file-download-options '{"saveAs": true}' --browserwindow-options '{"webPreferences":{"nativeWindowOpen":true,"affinity": "main-window"}}' --clear-cache
```
Log:
```
    Hi! Nativefier is minimally maintained these days, and needs more hands.
    If you have the time & motivation, help with bugfixes and maintenance is VERY welcome.
    Please go to https://github.com/nativefier/nativefier and help how you can. Thanks.



Processing options...

Preparing Electron app...

Converting icons...

Packaging... This will take a few seconds, maybe minutes if the requested Electron isn't cached yet...
Packaging app for platform linux x64 using electron v18.0.3

Finalizing build...
App built to /home/dkadioglu/OutlookWeb-linux-x64, move to wherever it makes sense for you and run the contained executable file (prefixing with ./ if necessary)
Menu/desktop shortcuts are up to you, because Nativefier cannot know where you're going to move the app. Search for "linux .desktop file" for help, or see https://wiki.archlinux.org/index.php/Desktop_entries
```
Build without issue:
Command:
```
nativefier -n "Outlook Web" -p linux -a x64 --tray https://some-owa-server.url --single-instance --internal-urls "(.*?)(xx\.yy\.zz)(.*?)"  --icon OutlookWeb.ico --inject OutlookWeb.js -m --file-download-options '{"saveAs": true}' --browserwindow-options '{"webPreferences":{"nativeWindowOpen":true,"affinity": "main-window"}}' --clear-cache --electron-version 17.4.0
```
Log:
```
    Hi! Nativefier is minimally maintained these days, and needs more hands.
    If you have the time & motivation, help with bugfixes and maintenance is VERY welcome.
    Please go to https://github.com/nativefier/nativefier and help how you can. Thanks.



Processing options...

ATTENTION: Using **old** Electron version 17.4.0 as requested. 
It's untested, bugs and horror will happen, you're on your own. 
Simply abort & re-run without passing the version flag to default to 18.0.3

Preparing Electron app...

Converting icons...

Packaging... This will take a few seconds, maybe minutes if the requested Electron isn't cached yet...
Packaging app for platform linux x64 using electron v17.4.0

Finalizing build...
App built to /home/dkadioglu/OutlookWeb-linux-x64, move to wherever it makes sense for you and run the contained executable file (prefixing with ./ if necessary)
Menu/desktop shortcuts are up to you, because Nativefier cannot know where you're going to move the app. Search for "linux .desktop file" for help, or see https://wiki.archlinux.org/index.php/Desktop_entries
```
**Expected behavior**

Attachments of an email are shown as preview in Outlook Web App.

**Actual behavior**

Attachments of an email are NOT shown as preview in Outlook Web App. Instead a popup is shown with the german error message:  Fehler beim Laden des PDF-Dokuments. Basically, a generic error message indicating no reason. In the console of Web Developer Tools is no difference in output between the build with 18.0.3 and 17.4.0.

**Debug info**

Console logs of your `nativefier` build command, with `--verbose` flag (I anonymized the urls, as this is the Exchange Server of my employer.).
electron 18:
```
Running in verbose mode! This will produce a mountain of logs and is recommended only for troubleshooting or if you like Shakespeare.


    Hi! Nativefier is minimally maintained these days, and needs more hands.
    If you have the time & motivation, help with bugfixes and maintenance is VERY welcome.
    Please go to https://github.com/nativefier/nativefier and help how you can. Thanks.



Processing options...
rawOptions {
  icon: 'OutlookWeb.ico',
  i: 'OutlookWeb.ico',
  _: [ 'https://some-exchange-server.url' ],
  n: 'Outlook Web',
  name: 'Outlook Web',
  p: 'linux',
  platform: 'linux',
  a: 'x64',
  arch: 'x64',
  tray: 'true',
  'single-instance': true,
  singleInstance: true,
  'internal-urls': '(.*?)(xx\\.yy\\.zz)(.*?)',
  internalUrls: '(.*?)(xx\\.yy\\.zz)(.*?)',
  inject: [ 'OutlookWeb.js' ],
  m: true,
  'show-menu-bar': true,
  showMenuBar: true,
  'file-download-options': { saveAs: true },
  fileDownloadOptions: { saveAs: true },
  'browserwindow-options': {
    webPreferences: { nativeWindowOpen: true, affinity: 'main-window' }
  },
  browserwindowOptions: {
    webPreferences: { nativeWindowOpen: true, affinity: 'main-window' }
  },
  'clear-cache': true,
  clearCache: true,
  verbose: true,
  c: false,
  conceal: false,
  'no-overwrite': false,
  noOverwrite: false,
  overwrite: true,
  portable: false,
  widevine: false,
  'always-on-top': false,
  alwaysOnTop: false,
  'disable-context-menu': false,
  disableContextMenu: false,
  'disable-dev-tools': false,
  disableDevTools: false,
  'full-screen': false,
  fullScreen: false,
  'hide-window-frame': false,
  hideWindowFrame: false,
  maximize: false,
  zoom: 1,
  'user-agent-honest': false,
  honest: false,
  userAgentHonest: false,
  'block-external-urls': false,
  blockExternalUrls: false,
  'strict-internal-urls': false,
  strictInternalUrls: false,
  'disable-gpu': false,
  disableGpu: false,
  'enable-es3-apis': false,
  enableEs3Apis: false,
  'ignore-gpu-blacklist': false,
  ignoreGpuBlacklist: false,
  'disable-old-build-warning-yesiknowitisinsecure': false,
  disableOldBuildWarningYesiknowitisinsecure: false,
  'ignore-certificate': false,
  ignoreCertificate: false,
  insecure: false,
  flash: false,
  bounce: false,
  counter: false,
  'darwin-dark-mode-support': false,
  darwinDarkModeSupport: false,
  f: false,
  'fast-quit': false,
  fastQuit: false,
  quiet: false,
  '$0': 'nativefier',
  targetUrl: 'https://some-exchange-server.url',
  out: undefined
}
Normalized URL https://some-exchange-server.url to: https://some-exchange-server.url/
Running in verbose mode! This will produce a mountain of logs and is recommended only for troubleshooting or if you like Shakespeare.

Performing async options post-processing.
Got icon from options. Using it, no inferring needed
Sanitized filename for Outlook Web : OutlookWeb
options {
  packager: {
    appCopyright: undefined,
    appVersion: undefined,
    arch: 'x64',
    asar: false,
    buildVersion: undefined,
    darwinDarkModeSupport: false,
    dir: '/usr/lib/node_modules/nativefier/app',
    electronVersion: '18.0.3',
    icon: 'OutlookWeb.ico',
    name: 'OutlookWeb',
    out: '/home/dkadioglu',
    overwrite: true,
    quiet: false,
    platform: 'linux',
    portable: false,
    targetUrl: 'https://some-exchange-server.url/',
    tmpdir: false,
    upgrade: false,
    upgradeFrom: undefined,
    win32metadata: {
      ProductName: 'Outlook Web',
      InternalName: 'Outlook Web',
      FileDescription: 'Outlook Web'
    }
  },
  nativefier: {
    accessibilityPrompt: true,
    alwaysOnTop: false,
    backgroundColor: undefined,
    basicAuthPassword: undefined,
    basicAuthUsername: undefined,
    blockExternalUrls: false,
    bookmarksMenu: undefined,
    bounce: false,
    browserwindowOptions: { webPreferences: [Object] },
    clearCache: true,
    counter: false,
    crashReporter: undefined,
    disableContextMenu: false,
    disableDevTools: false,
    disableGpu: false,
    diskCacheSize: undefined,
    disableOldBuildWarning: false,
    enableEs3Apis: false,
    fastQuit: false,
    fileDownloadOptions: { saveAs: true },
    flashPluginDir: undefined,
    fullScreen: false,
    globalShortcuts: undefined,
    hideWindowFrame: false,
    ignoreCertificate: false,
    ignoreGpuBlacklist: false,
    inject: [ 'OutlookWeb.js' ],
    insecure: false,
    internalUrls: '(.*?)(xx\\.yy\\.zz)(.*?)',
    lang: undefined,
    maximize: false,
    nativefierVersion: '47.0.0',
    quiet: false,
    processEnvs: undefined,
    proxyRules: undefined,
    showMenuBar: true,
    singleInstance: true,
    strictInternalUrls: false,
    titleBarStyle: undefined,
    tray: 'true',
    userAgent: undefined,
    userAgentHonest: false,
    verbose: true,
    versionString: undefined,
    width: 1280,
    height: 800,
    minWidth: undefined,
    minHeight: undefined,
    maxWidth: undefined,
    maxHeight: undefined,
    widevine: false,
    x: undefined,
    y: undefined,
    zoom: 1
  }
}

Preparing Electron app...
Copying electron app from /usr/lib/node_modules/nativefier/app to /tmp/nativefier-10-48-12-app--348362-3Pg9dPXQHI6C
Writing app config to /tmp/nativefier-10-48-12-app--348362-3Pg9dPXQHI6C/nativefier.json {
  accessibilityPrompt: true,
  alwaysOnTop: false,
  appBundleId: undefined,
  appCategoryType: undefined,
  appCopyright: undefined,
  appVersion: undefined,
  arch: 'x64',
  asar: false,
  backgroundColor: undefined,
  basicAuthPassword: undefined,
  basicAuthUsername: undefined,
  blockExternalUrls: false,
  bounce: false,
  browserwindowOptions: {
    webPreferences: { nativeWindowOpen: true, affinity: 'main-window' }
  },
  buildDate: 1650790092933,
  buildVersion: undefined,
  clearCache: true,
  counter: false,
  crashReporter: undefined,
  darwinDarkModeSupport: false,
  derefSymlinks: undefined,
  disableContextMenu: false,
  disableDevTools: false,
  disableGpu: false,
  disableOldBuildWarning: false,
  diskCacheSize: undefined,
  download: undefined,
  electronVersionUsed: '18.0.3',
  enableEs3Apis: false,
  executableName: undefined,
  fastQuit: false,
  fileDownloadOptions: { saveAs: true },
  flashPluginDir: undefined,
  fullScreen: false,
  globalShortcuts: undefined,
  height: 800,
  helperBundleId: undefined,
  hideWindowFrame: false,
  ignoreCertificate: false,
  ignoreGpuBlacklist: false,
  insecure: false,
  internalUrls: '(.*?)(xx\\.yy\\.zz)(.*?)',
  isUpgrade: false,
  junk: undefined,
  lang: undefined,
  maximize: false,
  maxHeight: undefined,
  maxWidth: undefined,
  minHeight: undefined,
  minWidth: undefined,
  name: 'OutlookWeb',
  nativefierVersion: '47.0.0',
  osxNotarize: undefined,
  osxSign: undefined,
  portable: false,
  processEnvs: undefined,
  protocols: undefined,
  proxyRules: undefined,
  prune: undefined,
  quiet: false,
  showMenuBar: true,
  singleInstance: true,
  strictInternalUrls: false,
  targetUrl: 'https://some-exchange-server.url/',
  titleBarStyle: undefined,
  tray: 'true',
  usageDescription: undefined,
  userAgent: undefined,
  userAgentHonest: false,
  versionString: undefined,
  width: 1280,
  widevine: false,
  win32metadata: {
    ProductName: 'Outlook Web',
    InternalName: 'Outlook Web',
    FileDescription: 'Outlook Web'
  },
  x: undefined,
  y: undefined,
  zoom: 1,
  oldBuildWarningText: ''
}
Copying 1 files to inject in app.
Copying injection file "OutlookWeb.js" to "/tmp/nativefier-10-48-12-app--348362-3Pg9dPXQHI6C/inject/inject-ab8eaa.js"
Updating /tmp/nativefier-10-48-12-app--348362-3Pg9dPXQHI6C/package.json 'name' field to outlookweb-nativefier-393d50

Converting icons...
Converting icon OutlookWeb.ico to /tmp/nativefier-10-48-12-iconconv--348362-hQXG4aDO5VgK/icon.png. Calling shell command: "/usr/lib/node_modules/nativefier/icon-scripts/convertToPng" "OutlookWeb.ico" "/tmp/nativefier-10-48-12-iconconv--348362-hQXG4aDO5VgK/icon.png"
Conversion succeeded and produced icon at /tmp/nativefier-10-48-12-iconconv--348362-hQXG4aDO5VgK/icon.png
Copying icons if necessary
Copying icon /tmp/nativefier-10-48-12-iconconv--348362-hQXG4aDO5VgK/icon.png to /tmp/nativefier-10-48-12-app--348362-3Pg9dPXQHI6C/icon.png

Packaging... This will take a few seconds, maybe minutes if the requested Electron isn't cached yet...
  electron-packager Electron Packager 15.4.0
  electron-packager Node v16.14.2
  electron-packager Host Operating system: linux 5.17.1-zen1-1-zen (x64) +0ms
  electron-packager Packager Options: {"arch":"x64","asar":false,"darwinDarkModeSupport":false,"dir":"/tmp/nativefier-10-48-12-app--348362-3Pg9dPXQHI6C","electronVersion":"18.0.3","icon":"/tmp/nativefier-10-48-12-iconconv--348362-hQXG4aDO5VgK/icon.png","name":"OutlookWeb","out":"/home/dkadioglu","overwrite":true,"quiet":false,"platform":"linux","portable":false,"targetUrl":"https://some-exchange-server.url/","tmpdir":false,"upgrade":false,"win32metadata":{"ProductName":"Outlook Web","InternalName":"Outlook Web","FileDescription":"Outlook Web"},"appBundleId":"com.electron.nativefier.outlookweb-nativefier-393d50"} +1ms
  electron-packager Target Platforms: linux +0ms
  electron-packager Target Architectures: x64 +0ms
  electron-packager Inferring appVersion from version in /tmp/nativefier-10-48-12-app--348362-3Pg9dPXQHI6C/package.json +0ms
  electron-packager Application name: OutlookWeb +5ms
  electron-packager Target Electron version: 18.0.3 +0ms
  electron-packager Ignored path regular expressions: [
  '/package-lock\\.json$',
  '/yarn\\.lock$',
  '/\\.git($|/)',
  '/node_modules/\\.bin($|/)',
  '\\.o(bj)?$',
  '/tmp/electron-packager'
] +0ms
  electron-packager Downloading Electron with options {"platform":"linux","arch":"x64","version":"18.0.3","artifactName":"electron"} +0ms
Packaging app for platform linux x64 using electron v18.0.3
  electron-packager Creating /home/dkadioglu/linux-x64-template +4ms
  electron-packager Extracting /home/dkadioglu/.cache/electron/f4b53fda321adc66035985eac16777325dc589e9f8d2c60f8a36c64dccc7a9c0/electron-v18.0.3-linux-x64.zip to /home/dkadioglu/linux-x64-template +0ms
  electron-packager Initializing app in /home/dkadioglu/OutlookWeb-linux-x64 from /home/dkadioglu/linux-x64-template template +0ms
  electron-packager Ignored paths based on the out param: [
  '/home/dkadioglu/OutlookWeb-darwin-x64',
  '/home/dkadioglu/OutlookWeb-darwin-arm64',
  '/home/dkadioglu/OutlookWeb-linux-ia32',
  '/home/dkadioglu/OutlookWeb-linux-x64',
  '/home/dkadioglu/OutlookWeb-linux-armv7l',
  '/home/dkadioglu/OutlookWeb-linux-arm64',
  '/home/dkadioglu/OutlookWeb-linux-mips64el',
  '/home/dkadioglu/OutlookWeb-mas-x64',
  '/home/dkadioglu/OutlookWeb-mas-arm64',
  '/home/dkadioglu/OutlookWeb-win32-ia32',
  '/home/dkadioglu/OutlookWeb-win32-x64',
  '/home/dkadioglu/OutlookWeb-win32-arm64'
] +2s
  electron-packager Validating bundled Electron app +0ms
  electron-packager Checking for a package.json file +0ms
  electron-packager Checking for the main entry point file +0ms
  electron-packager Validation complete +1ms
  electron-packager Renaming electron to OutlookWeb in /home/dkadioglu/OutlookWeb-linux-x64 +40ms

Finalizing build...
App built to /home/dkadioglu/OutlookWeb-linux-x64, move to wherever it makes sense for you and run the contained executable file (prefixing with ./ if necessary)
Menu/desktop shortcuts are up to you, because Nativefier cannot know where you're going to move the app. Search for "linux .desktop file" for help, or see https://wiki.archlinux.org/index.php/Desktop_entries
````
electron 17.4.0:
```
Running in verbose mode! This will produce a mountain of logs and is recommended only for troubleshooting or if you like Shakespeare.


    Hi! Nativefier is minimally maintained these days, and needs more hands.
    If you have the time & motivation, help with bugfixes and maintenance is VERY welcome.
    Please go to https://github.com/nativefier/nativefier and help how you can. Thanks.



Processing options...
rawOptions {
  icon: 'OutlookWeb.ico',
  i: 'OutlookWeb.ico',
  _: [ 'https://some-exchange-server.url' ],
  n: 'Outlook Web',
  name: 'Outlook Web',
  p: 'linux',
  platform: 'linux',
  a: 'x64',
  arch: 'x64',
  tray: 'true',
  'single-instance': true,
  singleInstance: true,
  'internal-urls': '(.*?)(xx\\.yy\\.zz)(.*?)',
  internalUrls: '(.*?)(xx\\.yy\\.zz)(.*?)',
  inject: [ 'OutlookWeb.js' ],
  m: true,
  'show-menu-bar': true,
  showMenuBar: true,
  'file-download-options': { saveAs: true },
  fileDownloadOptions: { saveAs: true },
  'browserwindow-options': {
    webPreferences: { nativeWindowOpen: true, affinity: 'main-window' }
  },
  browserwindowOptions: {
    webPreferences: { nativeWindowOpen: true, affinity: 'main-window' }
  },
  'clear-cache': true,
  clearCache: true,
  'electron-version': '17.4.0',
  e: '17.4.0',
  electronVersion: '17.4.0',
  verbose: true,
  c: false,
  conceal: false,
  'no-overwrite': false,
  noOverwrite: false,
  overwrite: true,
  portable: false,
  widevine: false,
  'always-on-top': false,
  alwaysOnTop: false,
  'disable-context-menu': false,
  disableContextMenu: false,
  'disable-dev-tools': false,
  disableDevTools: false,
  'full-screen': false,
  fullScreen: false,
  'hide-window-frame': false,
  hideWindowFrame: false,
  maximize: false,
  zoom: 1,
  'user-agent-honest': false,
  honest: false,
  userAgentHonest: false,
  'block-external-urls': false,
  blockExternalUrls: false,
  'strict-internal-urls': false,
  strictInternalUrls: false,
  'disable-gpu': false,
  disableGpu: false,
  'enable-es3-apis': false,
  enableEs3Apis: false,
  'ignore-gpu-blacklist': false,
  ignoreGpuBlacklist: false,
  'disable-old-build-warning-yesiknowitisinsecure': false,
  disableOldBuildWarningYesiknowitisinsecure: false,
  'ignore-certificate': false,
  ignoreCertificate: false,
  insecure: false,
  flash: false,
  bounce: false,
  counter: false,
  'darwin-dark-mode-support': false,
  darwinDarkModeSupport: false,
  f: false,
  'fast-quit': false,
  fastQuit: false,
  quiet: false,
  '$0': 'nativefier',
  targetUrl: 'https://some-exchange-server.url',
  out: undefined
}
Normalized URL https://some-exchange-server.url to: https://some-exchange-server.url/
Running in verbose mode! This will produce a mountain of logs and is recommended only for troubleshooting or if you like Shakespeare.

ATTENTION: Using **old** Electron version 17.4.0 as requested. 
It's untested, bugs and horror will happen, you're on your own. 
Simply abort & re-run without passing the version flag to default to 18.0.3

Performing async options post-processing.
Got icon from options. Using it, no inferring needed
Sanitized filename for Outlook Web : OutlookWeb
options {
  packager: {
    appCopyright: undefined,
    appVersion: undefined,
    arch: 'x64',
    asar: false,
    buildVersion: undefined,
    darwinDarkModeSupport: false,
    dir: '/usr/lib/node_modules/nativefier/app',
    electronVersion: '17.4.0',
    icon: 'OutlookWeb.ico',
    name: 'OutlookWeb',
    out: '/home/dkadioglu',
    overwrite: true,
    quiet: false,
    platform: 'linux',
    portable: false,
    targetUrl: 'https://some-exchange-server.url/',
    tmpdir: false,
    upgrade: false,
    upgradeFrom: undefined,
    win32metadata: {
      ProductName: 'Outlook Web',
      InternalName: 'Outlook Web',
      FileDescription: 'Outlook Web'
    }
  },
  nativefier: {
    accessibilityPrompt: true,
    alwaysOnTop: false,
    backgroundColor: undefined,
    basicAuthPassword: undefined,
    basicAuthUsername: undefined,
    blockExternalUrls: false,
    bookmarksMenu: undefined,
    bounce: false,
    browserwindowOptions: { webPreferences: [Object] },
    clearCache: true,
    counter: false,
    crashReporter: undefined,
    disableContextMenu: false,
    disableDevTools: false,
    disableGpu: false,
    diskCacheSize: undefined,
    disableOldBuildWarning: false,
    enableEs3Apis: false,
    fastQuit: false,
    fileDownloadOptions: { saveAs: true },
    flashPluginDir: undefined,
    fullScreen: false,
    globalShortcuts: undefined,
    hideWindowFrame: false,
    ignoreCertificate: false,
    ignoreGpuBlacklist: false,
    inject: [ 'OutlookWeb.js' ],
    insecure: false,
    internalUrls: '(.*?)(xx\\.yy\\.zz)(.*?)',
    lang: undefined,
    maximize: false,
    nativefierVersion: '47.0.0',
    quiet: false,
    processEnvs: undefined,
    proxyRules: undefined,
    showMenuBar: true,
    singleInstance: true,
    strictInternalUrls: false,
    titleBarStyle: undefined,
    tray: 'true',
    userAgent: undefined,
    userAgentHonest: false,
    verbose: true,
    versionString: undefined,
    width: 1280,
    height: 800,
    minWidth: undefined,
    minHeight: undefined,
    maxWidth: undefined,
    maxHeight: undefined,
    widevine: false,
    x: undefined,
    y: undefined,
    zoom: 1
  }
}

Preparing Electron app...
Copying electron app from /usr/lib/node_modules/nativefier/app to /tmp/nativefier-11-1-43-app--348932-SDedlpThw2Xl
Writing app config to /tmp/nativefier-11-1-43-app--348932-SDedlpThw2Xl/nativefier.json {
  accessibilityPrompt: true,
  alwaysOnTop: false,
  appBundleId: undefined,
  appCategoryType: undefined,
  appCopyright: undefined,
  appVersion: undefined,
  arch: 'x64',
  asar: false,
  backgroundColor: undefined,
  basicAuthPassword: undefined,
  basicAuthUsername: undefined,
  blockExternalUrls: false,
  bounce: false,
  browserwindowOptions: {
    webPreferences: { nativeWindowOpen: true, affinity: 'main-window' }
  },
  buildDate: 1650790903528,
  buildVersion: undefined,
  clearCache: true,
  counter: false,
  crashReporter: undefined,
  darwinDarkModeSupport: false,
  derefSymlinks: undefined,
  disableContextMenu: false,
  disableDevTools: false,
  disableGpu: false,
  disableOldBuildWarning: false,
  diskCacheSize: undefined,
  download: undefined,
  electronVersionUsed: '17.4.0',
  enableEs3Apis: false,
  executableName: undefined,
  fastQuit: false,
  fileDownloadOptions: { saveAs: true },
  flashPluginDir: undefined,
  fullScreen: false,
  globalShortcuts: undefined,
  height: 800,
  helperBundleId: undefined,
  hideWindowFrame: false,
  ignoreCertificate: false,
  ignoreGpuBlacklist: false,
  insecure: false,
  internalUrls: '(.*?)(xx\\.yy\\.zz)(.*?)',
  isUpgrade: false,
  junk: undefined,
  lang: undefined,
  maximize: false,
  maxHeight: undefined,
  maxWidth: undefined,
  minHeight: undefined,
  minWidth: undefined,
  name: 'OutlookWeb',
  nativefierVersion: '47.0.0',
  osxNotarize: undefined,
  osxSign: undefined,
  portable: false,
  processEnvs: undefined,
  protocols: undefined,
  proxyRules: undefined,
  prune: undefined,
  quiet: false,
  showMenuBar: true,
  singleInstance: true,
  strictInternalUrls: false,
  targetUrl: 'https://some-exchange-server.url/',
  titleBarStyle: undefined,
  tray: 'true',
  usageDescription: undefined,
  userAgent: undefined,
  userAgentHonest: false,
  versionString: undefined,
  width: 1280,
  widevine: false,
  win32metadata: {
    ProductName: 'Outlook Web',
    InternalName: 'Outlook Web',
    FileDescription: 'Outlook Web'
  },
  x: undefined,
  y: undefined,
  zoom: 1,
  oldBuildWarningText: ''
}
Copying 1 files to inject in app.
Copying injection file "OutlookWeb.js" to "/tmp/nativefier-11-1-43-app--348932-SDedlpThw2Xl/inject/inject-dbfc70.js"
Updating /tmp/nativefier-11-1-43-app--348932-SDedlpThw2Xl/package.json 'name' field to outlookweb-nativefier-393d50

Converting icons...
Converting icon OutlookWeb.ico to /tmp/nativefier-11-1-43-iconconv--348932-haICSyPFn4Vq/icon.png. Calling shell command: "/usr/lib/node_modules/nativefier/icon-scripts/convertToPng" "OutlookWeb.ico" "/tmp/nativefier-11-1-43-iconconv--348932-haICSyPFn4Vq/icon.png"
Conversion succeeded and produced icon at /tmp/nativefier-11-1-43-iconconv--348932-haICSyPFn4Vq/icon.png
Copying icons if necessary
Copying icon /tmp/nativefier-11-1-43-iconconv--348932-haICSyPFn4Vq/icon.png to /tmp/nativefier-11-1-43-app--348932-SDedlpThw2Xl/icon.png

Packaging... This will take a few seconds, maybe minutes if the requested Electron isn't cached yet...
  electron-packager Electron Packager 15.4.0
  electron-packager Node v16.14.2
  electron-packager Host Operating system: linux 5.17.1-zen1-1-zen (x64) +0ms
  electron-packager Packager Options: {"arch":"x64","asar":false,"darwinDarkModeSupport":false,"dir":"/tmp/nativefier-11-1-43-app--348932-SDedlpThw2Xl","electronVersion":"17.4.0","icon":"/tmp/nativefier-11-1-43-iconconv--348932-haICSyPFn4Vq/icon.png","name":"OutlookWeb","out":"/home/dkadioglu","overwrite":true,"quiet":false,"platform":"linux","portable":false,"targetUrl":"https://some-exchange-server.url/","tmpdir":false,"upgrade":false,"win32metadata":{"ProductName":"Outlook Web","InternalName":"Outlook Web","FileDescription":"Outlook Web"},"appBundleId":"com.electron.nativefier.outlookweb-nativefier-393d50"} +1ms
  electron-packager Target Platforms: linux +0ms
  electron-packager Target Architectures: x64 +0ms
  electron-packager Inferring appVersion from version in /tmp/nativefier-11-1-43-app--348932-SDedlpThw2Xl/package.json +0ms
  electron-packager Application name: OutlookWeb +5ms
  electron-packager Target Electron version: 17.4.0 +0ms
  electron-packager Ignored path regular expressions: [
  '/package-lock\\.json$',
  '/yarn\\.lock$',
  '/\\.git($|/)',
  '/node_modules/\\.bin($|/)',
  '\\.o(bj)?$',
  '/tmp/electron-packager'
] +0ms
  electron-packager Downloading Electron with options {"platform":"linux","arch":"x64","version":"17.4.0","artifactName":"electron"} +0ms
Packaging app for platform linux x64 using electron v17.4.0
  electron-packager Creating /home/dkadioglu/linux-x64-template +4ms
  electron-packager Extracting /home/dkadioglu/.cache/electron/6ba1e83a63c01ce05e10985bfc4d955c6ad43a3d135b25833fffe94983c2e3a8/electron-v17.4.0-linux-x64.zip to /home/dkadioglu/linux-x64-template +0ms
  electron-packager Initializing app in /home/dkadioglu/OutlookWeb-linux-x64 from /home/dkadioglu/linux-x64-template template +0ms
  electron-packager Ignored paths based on the out param: [
  '/home/dkadioglu/OutlookWeb-darwin-x64',
  '/home/dkadioglu/OutlookWeb-darwin-arm64',
  '/home/dkadioglu/OutlookWeb-linux-ia32',
  '/home/dkadioglu/OutlookWeb-linux-x64',
  '/home/dkadioglu/OutlookWeb-linux-armv7l',
  '/home/dkadioglu/OutlookWeb-linux-arm64',
  '/home/dkadioglu/OutlookWeb-linux-mips64el',
  '/home/dkadioglu/OutlookWeb-mas-x64',
  '/home/dkadioglu/OutlookWeb-mas-arm64',
  '/home/dkadioglu/OutlookWeb-win32-ia32',
  '/home/dkadioglu/OutlookWeb-win32-x64',
  '/home/dkadioglu/OutlookWeb-win32-arm64'
] +2s
  electron-packager Validating bundled Electron app +0ms
  electron-packager Checking for a package.json file +0ms
  electron-packager Checking for the main entry point file +1ms
  electron-packager Validation complete +0ms
  electron-packager Renaming electron to OutlookWeb in /home/dkadioglu/OutlookWeb-linux-x64 +43ms

Finalizing build...
App built to /home/dkadioglu/OutlookWeb-linux-x64, move to wherever it makes sense for you and run the contained executable file (prefixing with ./ if necessary)
Menu/desktop shortcuts are up to you, because Nativefier cannot know where you're going to move the app. Search for "linux .desktop file" for help, or see https://wiki.archlinux.org/index.php/Desktop_entries
```
Screenshots:
electron 18:
![image](https://user-images.githubusercontent.com/1633773/164968665-aa76a703-adc9-4d6b-a90e-d8913b5221c1.png)
electron 17.4.0:
![image](https://user-images.githubusercontent.com/1633773/164969004-fe1798b2-3bca-4f99-8097-d21a1780c4b0.png)

**Context**

 - Nativefier: 47.0.0
 - Node.js: 16.14.2
 - Npm: 8.5.5
 - OS: Arch Linux, x86_64, fully updated
 - Is it a regression? Yes. 47.0.0 works only with electron <18

**Comment by dkadioglu** [comment]
This seems to be fixed with electron 18.2.0, see https://github.com/electron/electron/pull/33711

**Comment by ronjouch** [comment]
> This seems to be fixed with electron 18.2.0, see [electron/electron#33711](https://github.com/electron/electron/pull/33711)

@dkadioglu neat, thx for the ping. Closing as freshly-released Nativefier 47.1.0 will default to it.
